src/Tickets/Commerce/Gateways/Stripe/Client.php

```diff
@@ -34,6 +35,15 @@ class Client {
 	 */
 	public $payment_intent_transient_name;
 
+	/**
+	 * The percentage applied to Stripe transactions. Currently set at 2%.
+	 *
+	 * @since TBD
+	 *
+	 * @var float
+	 */
+	public $application_fee_percentage = 0.02;
```

Comment:
Can we move this to static protected value and use a static public method to pull it?